prime-router/docs/design/proposals/translation-template-test-plan.md

```diff
@@ -0,0 +1,57 @@
+# Translation Template Test Plan
+
+## Problem Statement
+As a Reportstream engineer, I want tests that verify that the changes I make to translations will not break essential 
+rules set forth by the spec or our users. 
+
+Currently, we use integration tests for this purpose, but these do not give us the ability to easily tell which thing 
```

Comment:
I think this is partially true, but for ELIMS there isn't any special handling for them, so the integration tests are just loosely checking that for there messages our source code can do HL7->FHIR->HL7